docs/source/api/notebooks/Notebook_5_GP_solution.ipynb

```diff
@@ -0,0 +1,631 @@
+    "        decoder = EntityClassifier(in_dim=hid_size,\n",
+    "                                   num_classes=num_classes,\n",
+    "                                   multilabel=multilabel)\n",
+    "        self.set_decoder(decoder)\n",
+    "\n",
+    "        # classification loss function\n",
+    "        self.set_loss_func(ClassifyLossFunc(multilabel=multilabel))\n",
+    "\n",
+    "        # initialize model's optimizer\n",
+    "        self.init_optimizer(lr=0.001,\n",
+    "                            sparse_optimizer_lr=0.001,\n",
+    "                            weight_decay=0)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "model = RgcnNCModel4SuperOgbg(g=ogbg_data.g,\n",
+    "                              num_hid_layers=3,\n",
+    "                              node_feat_field=nfeats_4_modeling,\n",
+    "                              hid_size=128,\n",
+    "                              num_classes=2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# setup a classification evaluator for the trainer\n",
+    "evaluator = gs.eval.GSgnnClassificationEvaluator(eval_frequency=100,\n",
+    "                                                 eval_metric_list=['roc_auc'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# create a GraphStorm node task trainer for the RGCN model\n",
+    "trainer = gs.trainer.GSgnnNodePredictionTrainer(model)\n",
+    "trainer.setup_evaluator(evaluator)\n",
+    "trainer.setup_device(gs.utils.get_device())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Train the model with the trainer using fit() function\n",
+    "trainer.fit(train_loader=train_dataloader,\n",
+    "            val_loader=val_dataloader,\n",
+    "            test_loader=test_dataloader,\n",
+    "            num_epochs=50,\n",
+    "            save_model_path='a_save_path/')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we examine the model performance over the training process"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Extract accuracies from the trainer's evaluator:\n",
+    "val_accs, test_accs = [], []\n",
+    "for val_acc, test_acc in trainer.evaluator.history:\n",
+    "    val_accs.append(val_acc['roc_auc'])\n",
+    "    test_accs.append(test_acc['roc_auc'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
```

Comment:
Need to ensure matplotlib is installed, add to prerequisites for the user's env

Reply:
Yes, updated `Prerequisites` include this library.